### DISCUSSION TITLE
strings question.

### DISCUSSION THREAD

**Discussion by KX5-jeron** [Lailloken/Exile-UI#445] [Q&A]
Is there a way to toggle different string use cases on and off or even between them without going into the settings?? Main reason is for the inventory screen. I have different use cases for map rolling, general inventory searching, etc.. Is there a way to toggle between the different use cases while in the inventory?? Its not a huge deal to toggle them on and off manually but it would be nice if the context menu could change to the next relevant use case used with the inventory with a key bind..

**Comment by Lailloken** [comment]
No, there's currently no way to switch between use-cases on the fly. The feature activates whichever use-case it detects first (it goes through the list alphabetically).

I don't use the feature for stash searches, only for NPCs or benches, so it might not be fully optimized for that right now. What you can try is calibrate your use-cases with more specific parts of the stash, for example:

![image](https://github.com/user-attachments/assets/0700ac07-d824-4203-9d97-d7f1f935040d)

If you color-code your stashes correctly, it should always show the correct context-menu for the current tab. During calibration, if you screen-cap using right-click, you can move the snipping-panel with WASD for more precision.
***
I personally would only create a single "stash" use-case and calibrate it with the plaque at the top ( ![image](https://github.com/user-attachments/assets/26e34ba4-bbfc-4d68-aec2-22823a4942c8) ) and add tagged entries, e.g.:
- maps: bricks

- general: XYZ

- etc.

But that only works if you're not going to use that many strings.

**Comment by KX5-jeron** [comment]
Ok thank you for the detailed reply.. I will do it like you mentioned as it probably is better than I am doing currently..

different question, is it already in and I can't find it, or do you plan to add any functionality to make chat keybinds?? i run an app for only one purpose and that is to be able to hit F5 to send me to my hideout using the chat /hideout command..

**Reply by Lailloken** [reply]
There is support for that, but it's currently undocumented. Basically, you can create an additional `hotkeys custom.ahk` file in the `modules` folder and that will be treated as an "extension" to the tool. There are two ways to create the file:

1. open the `modules` folder within the tool-folder, then right-click into the folder, go to "new", click "AutoHotkey Script", and name the file `hotkeys custom`

  ![image](https://github.com/user-attachments/assets/b8c37384-262d-4bd4-99b1-fb998a01195b)

2. make a copy of any file in the `modules` folder, then rename the copy to `hotkeys custom`
***

Right-click the new file, click "edit" to open it in notepad. Delete everything inside the file, then paste this
```
#IfWinActive ahk_group poe_window

F5::
Clipboard := "/hideout"
ClipWait, 0.1
SendInput, {Enter}^{v}{Enter}
KeyWait, F5
Return

#If
```

Save the file, then restart the tool.

**Reply by KX5-jeron** [reply]
ahh ok.. can i do mulitiple?? I currently use F5= /hideout, F6= /guild hideout, and F7= /kingsmarch. Can I just make 3 separate files named hotkeys custom 1, hotkeys custom 2, hotkeys custom 3????

**Reply by KX5-jeron** [reply]
nevermind i added them all into the one hotkeys custom file and it works perfect!! Thanks for your help.
